dali/test/python/test_audiodecoder_spectogram.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvidia.dali.pipeline import Pipeline
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+import nvidia.dali as dali
+import numpy as np
+from numpy.testing import assert_array_equal, assert_allclose
+from functools import partial
+from test_utils import check_batch
+from test_utils import compare_pipelines
+from test_utils import get_dali_extra_path
+import librosa as librosa
+import math
+import os
+
+dali_extra = get_dali_extra_path()
+audio_files = os.path.join(dali_extra, "db", "audio")
+
+
+class SpectrogramPipeline(Pipeline):
+    def __init__(self, batch_size, nfft, window_length, window_step,
+                 num_threads=1, device_id=0):
+        super(SpectrogramPipeline, self).__init__(batch_size, num_threads, device_id)
+        self.input = ops.FileReader(device="cpu", file_root=audio_files)
+        self.decode = ops.AudioDecoder(device="cpu", dtype=types.FLOAT, downmix=True)
+        self.fft = ops.Spectrogram(device="cpu",
+                                   nfft=nfft,
+                                   window_length=window_length,
+                                   window_step=window_step,
+                                   power=2)
+
+    def define_graph(self):
+        read, _ = self.input()
+        audio, rate = self.decode(read)
+        spec = self.fft(audio)
+        return spec
```

Comment:
Can you also return the data from `decode` and validate their shapes?

Reply:
That's already tested, [here](https://github.com/NVIDIA/DALI/blob/master/dali/test/python/test_operator_audio_decoder.py)